source/utilities.cc

```diff
@@ -888,16 +890,92 @@ namespace aspect
           //Check to see if the prm file will be reading data from the disk or
 		// from a provided URL
 		if (readUrl) {
-			std::cout << "TESTING THE READ FROM URL == true" << std::endl;
+			std::cout << "TESTING THE READ FROM URL == true <<<<<<<< file: " << filename << std::endl;
```

Comment:
we should remove this - it could be wrapped in libdap's DBG() macro. Also, if it writes to cerr and not cout, the stuff gets output with less buffering/lag.